clients/include/testing_sbmv_batched.hpp

```diff
@@ -28,13 +28,10 @@ hipblasStatus_t testing_sbmv_batched(const Arguments& argus)
     int incy = argus.incy;
 
     int A_size = lda * M;
```

Comment:
I think others are now supporting large index so could change to 
size_t A_size = size_t(lda) * M ;   
Maybe fix now?

Reply:
Updated to size_t across the board now.